### PR TITLE
Added task to enable interfaces

### PR DESCRIPTION
**Pull request by mtannous**
Added a task into this graph in order to enable interfaces

### CODE REVIEW

**Comment by benbp** [comment]
Please squash and rebase the commits. 👍 otherwise

**Comment by mtannous** [comment]
All done.

File: lib/graphs/discovery-graph.js [hunk 1]
```diff
@@ -38,10 +38,18 @@ module.exports = {
             ignoreFailure: true
         },
         {
+            label: 'set-interfaces',
+            taskName: 'Task.Set.Interfaces',
+            waitOn: {
+                'catalog-bmc': 'finished'
+        },
```
Comment (anhou) [inline lib/graphs/discovery-graph.js]: indent for this curly brace

**Comment by anhou** [comment]
:+1:  with minor comment

**Comment by yyscamper** [comment]
:+1: after fix andrew's comment.

**Comment by yyscamper** [comment]
jenkins: ok to test

**Comment by PengTian0** [comment]
test this please

**Comment by PengTian0** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by PengTian0** [comment]
Can one of the admins verify this patch?

**Comment by PengTian0** [comment]
test this please

**Comment by PengTian0** [comment]
test this please

**Comment by geoff-reid** [comment]
jenkins: test this please

**Comment by mtannous** [comment]
test this please

**Comment by tannoa2** [comment]
jenkins can you please test this please for @mtannous

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Pull request merged**